### PR TITLE
issue #27 and #23 missing translation in devise and single qnotes in …

### PR DESCRIPTION
**Pull request by kasiaekiert**
…helpers

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -17,5 +17,5 @@ a.nav-link {
  }
 
  .flash_message {
-   margin-top: 30px;
+   margin-top: 0px;
```
[the remark this answers is not in the document]
Comment (womanonrails) [reply app/assets/stylesheets/main.scss]: I agree.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -2,10 +2,10 @@ module ApplicationHelper
   def flash_message
     flash.map do |key, value|
       level_class = case key
-        when 'notice' then "alert alert-info"
-        when 'success' then "alert alert-success"
-        when 'error' then "alert alert-danger"
-        when 'alert' then "alert alert-warning"
+        when 'notice' then 'alert alert-info'
```
Comment (womanonrails) [inline app/helpers/application_helper.rb]: When you have `case` then you don't need to do indent for `when`. You can think about it like in `if` and `else` they have the same indent. So the same rule is for `case` and `when`.

File: config/locales/devise.pl.yml [hunk 3]
```diff
@@ -30,6 +30,9 @@ pl:
       send_paranoid_instructions: "Jeżeli Twój adres istnieje w naszej bazie, otrzymasz za chwilę emaila z instrukcją zmiany hasła."
       updated: "Twoje hasło zostało zmienione. Zostałeś pomyślnie zalogowany."
       updated_not_active: "Twoje hasło zostało zmienione."
+      edit:
+        change_password: 'Zmień hasło'
+      new:
```
Comment (womanonrails) [inline config/locales/devise.pl.yml:35]: You forgot to add:
```
reset_password_instructions: 'Send me reset password instructions'
        devise:
          sessions:
            users:
              password:
                new: 'Forget the password?'
```
To polish version of translations.

**Comment by kasiaekiert** [comment]
Ok check it now please

File: app/assets/stylesheets/main.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .navbar {
-  margin-bottom: 0px !important;
+  margin-bottom: 0 !important;
```
Comment (womanonrails) [inline app/assets/stylesheets/main.scss]: This is not fixed. You still use `!important`

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -2,23 +2,22 @@ module ApplicationHelper
...
-        when 'notice' then "alert alert-info"
-        when 'success' then "alert alert-success"
-        when 'error' then "alert alert-danger"
-        when 'alert' then "alert alert-warning"
-      end 
+                      when 'notice' then 'alert alert-info'
+                      when 'success' then 'alert alert-success'
+                      when 'error' then 'alert alert-danger'
+                      when 'alert' then 'alert alert-warning'
+                    end
```
[the remark this answers is not in the document]
Comment (womanonrails) [reply app/helpers/application_helper.rb:9]: You have wrong white spaces and `when` and `end` should be in the same intend.

**Review by womanonrails — CHANGES_REQUESTED** [review]

**Pull request merged**